src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -1782,6 +1782,49 @@ DROP TABLE mx_test_schema_1.mx_table_1 CASCADE;
 DROP TABLE mx_testing_schema.mx_test_table;
 DROP TABLE mx_ref;
 DROP TABLE dist_table_1, dist_table_2;
+CREATE USER non_super_metadata_user;
```

Comment:
I'm guessing that this test would fail on Citus enterprise? Can you make sure to make it pass on both?